### PR TITLE
Remove `ipfs` binary from `nerdctl-full`

### LINKED ISSUE #3527: Consider removing `ipfs` binary from `nerdctl-full`
Enterprise companies often (blindly) ban installing any software that relates to "P2P".

Removing `ipfs` binary from `nerdctl-full` may help installing nerdctl on those environments.

Users can still opt-in to install `ipfs` from https://github.com/ipfs/kubo .

### PR DESCRIPTION
**Pull request by yankay**
Removing `ipfs` binary from `nerdctl-full`

fix #3527

### CODE REVIEW

**Comment by apostasie** [comment]
We need it for testing though.
We might want to start considering a more modular approach to this Dockerfile - and different targets for the published artifacts vs. the stuff we use for testing.

**Review by apostasie — APPROVED** [review]

File: Dockerfile
Comment (AkihiroSuda) [inline Dockerfile:1]: KUBO_VERSION in L39-40 should be moved to the “Test deps” section (L46-52)

Comment (yankay) [reply Dockerfile:1]: Thanks @AkihiroSuda 

It has been changed

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**